extensions/ql-vscode/src/view/common/DeterminateProgressRing.tsx

```diff
@@ -39,7 +39,7 @@ export const DeterminateProgressRing = ({ percent }: Props) => (
     aria-valuemax={100}
     aria-valuenow={percent}
   >
-    <svg className="progress" viewBox="0 0 16 16">
+    <svg className="progress" viewBox="0 0 16 16" role="presentation">
```

Comment:
Just out of curiosity, why a role and why 'presentation'?

Reply:
This fixes the [`github/a11y-svg-has-accessible-name`](https://github.com/github/eslint-plugin-github/blob/main/docs/rules/a11y-svg-has-accessible-name.md) ESLint rule. The `role="presentation"` basically just means that this content shouldn't really be read by screen readers and that they can ignore it. We don't need to add a title to the SVG because it already has `role="progressbar"`, so can already be recognized by screen readers.